docs/data-cookbooks/cnm-workflow.md

```diff
@@ -68,137 +68,171 @@ Provided the prerequisites have been fulfilled, you can begin adding the needed
+            ],
+            "ResultPath": "$.exception",
+            "Next": "CnmResponse"
+          }
+        ],
+        "Next": "SyncGranule"
+      },
+      "SyncGranule": {
+        "Parameters": {
+          "cma": {
+            "event.$": "$",
+            "task_config": {
+              "provider": "{$.meta.provider}",
+              "buckets": "{$.meta.buckets}",
+              "collection": "{$.meta.collection}",
+              "downloadBucket": "{$.meta.buckets.private.name}",
+              "stack": "{$.meta.stack}",
+              "cumulus_message": {
+                "outputs": [
+                  {
+                    "source": "{$.granules}",
+                    "destination": "{$.meta.input_granules}"
+                  },
+                  {
+                    "source": "{$}",
+                    "destination": "{$.payload}"
+                  }
+                ]
+              }
+            }
+          }
+        },
+        "Type": "Task",
+        "Resource": "${module.cumulus.sync_granule_task_lambda_function_arn}",
+        "Retry": [
+          {
+            "ErrorEquals": [
+              "States.ALL"
+            ],
+            "IntervalSeconds": 10,
+            "MaxAttempts": 3
+          }
+        ],
+        "Catch": [
+          {
+            "ErrorEquals": [
+              "States.ALL"
+            ],
+            "ResultPath": "$.exception",
+            "Next": "CnmResponse"
+          }
+        ],
+        "Next": "CnmResponse"
+      },
+      "CnmResponse": {
+        "Parameters": {
+          "cma": {
+            "event.$": "$",
+            "task_config": {
+              "OriginalCNM": "{$.meta.cnm}",
+              "CNMResponseStream": "ADD YOUR RESPONSE STREAM HERE",
+              "region": "us-east-1",
+              "WorkflowException": "{$.exception}",
+              "cumulus_message": {
+                "outputs": [
+                  {
+                    "source": "{$}",
+                    "destination": "{$.meta.cnmResponse}"
+                  }
+                ]
+              }
+            }
+          }
+        },
+        "Type": "Task",
+        "Resource": "${aws_lambda_function.cnm_response_task.arn}",
+        "Retry": [
+          {
+            "ErrorEquals": [
+              "States.ALL"
+            ],
+            "IntervalSeconds": 5,
+            "MaxAttempts": 3
+          }
+        ],
+        "End": true
+      }
+    }
+  }
+}
+JSON
 ```
 
 Again, please make sure to modify the value CNMResponseStream to match the stream name (not ARN) for your Kinesis response stream.
 
-### Task Configuration
+### Lambda Configuration
 
-The following tasks are required to be defined in the `lambdas.yml` configuration file.
+To execute this workflow, you're required to include several lambda resources in your deployment.  To do this, add the following task (lambda) definitions to your deployment along with the workflow you created above:
```

Comment:
Nit: can we change `lambda` to `Lambda` throughout?

Reply:
not a nit. 
b3cfd9534